packages/preferences-api/package.json

```diff
@@ -5,7 +5,9 @@
   "main": "lib/common/index.js",
   "typings": "lib/common/index.d.ts",
   "dependencies": {
-    "@theia/core": "^0.3.3"
+    "@theia/core": "^0.3.3",
+    "@types/clone": "^0.1.30",
```

Comment:
I would prefer to avoid adding new dependencies if one can achieve results with existing dependencies, like with phosphor